README.md

```diff
@@ -1,14 +1,4 @@
-[![Build Status](https://travis-ci.org/GaloisInc/bv-sized.svg?branch=master)](https://travis-ci.org/GaloisInc/bv-sized)
+# bv-sized
 
-bv-sized - A Haskell library for manipulating width-parameterized bitvectors
-===
-
-copyright (c) Ben Selfridge, Galois Inc. 2018
-
-This library defines a bitvector datatype that is parameterized by the
-vector width.
-
-Other information
-===
-
-* contact: benselfridge@galois.com
+This library defines a bitvector datatype that is parameterized by the vector
+width.
```

Comment:
I've usually heard the phrase "bit width" in relation to bitvectors more frequently than I've heard "vector width", so I'd prefer using the former phrasing.